pkg/nsx/services/vpc/store.go

```diff
@@ -49,7 +49,7 @@ func indexPathFunc(obj interface{}) ([]string, error) {
 var filterTag = func(v []model.Tag) []string {
 	res := make([]string, 0, 5)
 	for _, tag := range v {
-		if *tag.Scope == common.TagScopeVPCCRUID {
+		if *tag.Scope == common.TagScopeNetworkInfoCRUID {
```

Comment:
I don't think we should add NetworkInfo CR UID in VPC tag, there is a case that several NetworkInfo map to the same NSX VPC

Reply:
Yes, makes sense. I think we can remove the index, not used anymore.